UserManual.md

```diff
@@ -43,6 +43,44 @@ Both install packages will create a new `laminar` user and install (but not acti
 
 See the [development README](https://github.com/ohwgiles/laminar) for instructions for installing from source.
 
+## Building for Docker
+
+You can build an image that runs `laminard` by default, and contains `laminarc` for use based on `alpine:edge` using the `Dockerfile` in the `docker/` directory.
+
+Keep in mind that this is meant to be used as a base image to build from, so it contains only the minimum packages required to run laminar. The only shell available by default is sh and it does not even have ssh or git. You can use this image to run a basic build server, but it is recomended that you build a custom image from this base to better suit your needs.
+
+```bash
+# from the repository root:
+docker build [-t image:tag] -f docker/Dockerfile .
+```
+
+You can also build a docker image using a bash script provided in the root of the repository. You should already have docker installed on your machine for this to work.
+
+```bash
+# from the repository root:
+./docker-build.sh [(OPTIONAL) tag {default:latest}]
+```
+
+This will build an image from from the Dockerfile in `docker/` tagged as `laminar:${tag}`.
+
+### laminard
+
+The container will execute `laminard` by default. To start a laminar server with docker you can simply run the image as a daemon.
+
+```bash
+docker run -d --name laminar_server -p 8080:8080 [-v laminardir|laminar.conf] laminar:latest
+```
+
+You can customize laminar and persist your data by mounting your laminar directory to `/var/lib/laminar` and/or mounting a custom configuration file to `/etc/laminar.conf`. The uid:gid ownership for the files mounted to `/var/lib/laminar` should be `100:100`.
```

Comment:
Also by passing laminar configuration variables with `-e CONF=value`.

I don't like the whole uid:gid must be 100 thing. I would typically recommend having a user on the host, then launching `docker run` with the `-u` flag. Alternatively just remove that sentence and let the sysadmin figure out what's good for them